evaluation/edit_api_usecase.md

```diff
@@ -0,0 +1,68 @@
+## Edit An API
+### Primary Actor: Advanced User
+
+### Trigger
+The user wants to change an existing API which is linked to a given function.
+
+### Preconditions
+1. There is at least one (built-in or previously created via the Developer Suite) function available for the user's account.
+2. There is at least one (built-in or previously created via the Developer Suite) API to be editted available for the user's account.
+
+### Main success scenarion
+1. The user clicks on "Switch to Developer Suite".
+2. The user confirms the warning pop-up to enter the advanced section Developer Suite.
+3. The user switches to the second tab "Manage APIs".
+4. The user switches to the second sub-tab "Edit existing APIs".
+5. The user selects a target API by clicking on "Show Details".
+6. The user unlocks the protection mode to enter the edit mode.
+7. The user applies changes as far as fields are not protected.
+8. The user saves and finishes the API edit process.
+
+### Evaluation
+#### Task
+You, as an advanced user, are not satisfied with some information of the corona API for receiving latest news of corona confirmed cases.
+Edit an existing API based on the following information:
+
+In the list, select the API `COVID-19 data`, click on "Show Detail" and switch to edit mode.
+Some fields are not changeable in order to prevent errors by changing mandatory information which should be kept reliable consistent.
+Change the name of the API to `COVID-19 Cases` and set the priorty to `Medium`.
```

Comment:
Priorität auf Medium zurücksetzen geht nur, falls noch eine andere API für die Funktion existiert, da es immer eine preferred API geben muss, ist aber der Fall, da es noch die API Covid-19 gibt. (Vllt erklären, dass nun automatisch eine andere API mit möglichst hoher Prio als "preferred" ausgewählt wird, also in unserem Falle Covid-19.